validator/README.md

```diff
@@ -11,7 +11,23 @@ The following validations are applied to the entity definition files:
 
 For the summary and golden metrics definitions only the schema validation applies. 
 
-If you want to validate definition files manually, for example before opening a pull request, just put the file in a folder under `definitions` and run the validation locally as explained [here](#local-setup).
+If you want to validate definition files manually, for example before opening a pull request, just put the file in a folder under `definitions` and run the validation locally as explained [here](#running-via-docker).
+
+
+### Running via docker
+
+1. Install [Docker](https://www.docker.com/products/docker-desktop)
+2. Execute the following command:
+
+```
+docker-compose up --build --force-recreate validate-definitions
```

Comment:
```suggestion
docker-compose run --build --force-recreate validate-definitions
```

Reply:
I think that this doesn't work, but I can make it:

```
docker-compose run validate-definitions
```